magprime/__init__.py

```diff
@@ -42,6 +42,11 @@ class Attendee:
     def donation_cost(self):
         return self.extra_donation if self.extra_donation else 0
 
```

Comment:
You're right, this was broken (I didn't have test keys on my local copy to test it properly, though now I do).

As far as the need to do this goes, we have a field for the extra_donation that defaults to empty. So if the user leaves it empty and tries to pay, we're passing extra_donation as an empty string ("") which is how we were getting this problem.

I'm not sure this is the _best_ solution, though.